velox/docs/functions/spark/datetime.rst

```diff
@@ -82,7 +82,9 @@ These functions support TIMESTAMP and DATE input types.
 
     Adjusts ``unixTime`` (elapsed seconds since UNIX epoch) to configured session timezone, then
     converts it to a formatted time string according to ``format``. Only supports BIGINT type for
-    ``unixTime``.
+    ``unixTime``. Using `Simple <https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html>`
```

Comment:
I think it is important to clarify whether it uses Simple formatter in lenient or strict mode as this is not configurable.